src/mm-datepicker/mm-datepicker.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function () {
+
+	function _undef(a) {
+		return a !== null && a !== undefined && a !== "";
+	}
+
+	Polymer('mm-datepicker', {
+		ver:"<<version>>",
+		STATE_OPENED: "opened",
+		STATE_CLOSED: "closed",
+		publish: {
+
+			useTimezone: true,
+			useRange: true,
+			useTime: true,
+			useCommit: true,
+			resetOnClose: true,
+			dual: true,
+			dateFormat:"MM/DD/YYYY",
+			timeFormat:"hh:mm a",
+
+			rangeValue:"",
+			rangePresets: null,
+			rangeDescription:"Selected a Predefined Date Range",
+			timezoneDescription:"Select a Timezone",
+			timezones:null,
+			
+			start: null,
+			startEnabled: false,
+			startUserEnabled: true,
+			allowedStart: null,
+			startLabel:"start",
+			startEnabledLabel:null,
+			//internals
+			startDate: null,
+			startTime: null,
+			startTimePeriod:"am",
+			
+			end: null,
+			endEnabled: false,
+			endUserEnabled: true,
+			allowedEnd: null,
+			endLabel: "end",
+			endEnabledLabel:null,
+			//internals
+			endDate: null,
+			endTime: null,
+			endTimePeriod:"am",
+			
+			timezone: null,
+
+			state:'closed',
+			valign: { value: "bottom", reflect: true },
```

Comment:
valign default should also be "center" to avoid some gnarly positioning on 13" display